appveyor/install-miniconda.ps1

```diff
@@ -76,8 +76,12 @@ $env:PATH = "${env:PYTHON};${env:PYTHON}\Scripts;" + $env:PATH
 
 # Conda config
 conda config --set always_yes true
-conda config --add channels astropy-ci-extras
 
+if (! $env:CONDA_CHANNELS) {
+   conda config --add channels astropy astropy-ci-extras

```

Comment:
Me neither, but I never tried it either (Tom wrote this part for travis, so I guess he had reasons to loop over the channels). We'll see in 30 mins.